### PR TITLE
Remove unneeded includes in header files.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review
@baldersheim : FYI

### CODE REVIEW

**Review by baldersheim — APPROVED** [review]

File: searchsummary/src/vespa/searchsummary/docsummary/copy_dfw.h [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+#pragma once
+
+#include "docsum_field_writer.h"
+
+namespace search::docsummary {
+
+class ResultConfig;
+
+class CopyDFW : public DocsumFieldWriter
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/copy_dfw.h:11]: Consider adding class comment.

File: searchsummary/src/vespa/searchsummary/docsummary/empty_dfw.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "simple_dfw.h"
+
+namespace search::docsummary {
+
+class EmptyDFW : public SimpleDFW
```
Comment (geirst) [inline searchsummary/src/vespa/searchsummary/docsummary/empty_dfw.h:9]: Consider adding class comment.

**Review by geirst — APPROVED** [review]
👍

File: searchsummary/src/vespa/searchsummary/docsummary/copy_dfw.h (hunk 1 above)
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/copy_dfw.h:14]: Done.

File: searchsummary/src/vespa/searchsummary/docsummary/simple_dfw.h [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include "docsum_field_writer.h"
+
+namespace search::docsummary {
+
+class SimpleDFW : public DocsumFieldWriter
```
[the remark this answers is not in the document]
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/simple_dfw.h:13]: Done.

File: searchsummary/src/vespa/searchsummary/docsummary/empty_dfw.h (hunk 2 above)
Comment (toregge) [reply searchsummary/src/vespa/searchsummary/docsummary/empty_dfw.h:12]: Done.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**